README.md

```diff
@@ -28,18 +28,22 @@ Join the [Slack Developer Program](https://api.slack.com/developer-program) for
 
 Before you can run the app, you'll need to store some environment variables.
 
+
 1. Rename `.env.sample` to `.env`
-2. Open your apps setting page from
-   [this list](https://api.slack.com/apps), click _OAuth & Permissions_ in the
-   left hand menu, then copy the _Bot User OAuth Token_ into your `.env` file
-   under `SLACK_BOT_TOKEN`
-3. Click _Basic Information_ from the left hand menu and follow the steps in the
-   _App-Level Tokens_ section to create an app-level token with the
-   `connections:write` scope. Copy that token into your `.env` as
-   `SLACK_APP_TOKEN`.
-4. `OPENAI_API_KEY=YOUR_OPEN_API_KEY`
-
-If using Hugging Face - A fine grained [access token](https://huggingface.co/settings/tokens) with the "Make calls to Inference Providers" permission from Hugging Face is needed. Store this as `HUGGINGFACE_API_KEY`. Add `"@huggingface/inference": "^4.7.1",` to `package.json` under `dependencies`
+2. Open your apps setting page from [this list](https://api.slack.com/apps), click _OAuth & Permissions_ in the left hand menu, then copy the _Bot User OAuth Token_ into your `.env` file under `SLACK_BOT_TOKEN`
```

Comment:
📚 quibble: These numbered lines might be best to end with a period? Or the colon symbol when referencing a following code block?